test/big_query/podcasts_test.exs

```diff
@@ -5,36 +5,36 @@ defmodule Castle.BigQueryPodcastsTest do
 
   @tag :external
   test "lists podcasts" do
-    {result, _meta} = list(Timex.to_datetime(~D[2017-04-09]))
+    {result, _meta} = list(Timex.to_datetime(~D[2017-06-28]))
```

Comment:
These external tests are a bit wonky - they require actual BQ data to be present.  The only run if you `mix test --include external`, so not during normal CI.  But handy if you want to try things locally against live data.